src/main/java/fox/spiteful/avaritia/compat/botania/Tsundere.java

```diff
@@ -29,14 +30,17 @@ public static void baka() throws Compat.ItemNotFoundException {
 
         SubTileCheaty.lexicon = new LudicrousLexicon("asgardandelion", BotaniaAPI.categoryGenerationFlowers);
         SubTileCheaty.lexicon.addPage(BotaniaAPI.internalHandler.textPage("avaritia.lexicon.asgardandelion.0"));
-        SubTileCheaty.lexicon.setIcon(cheaty);
-
-        ExtremeCraftingManager.getInstance().addRecipe(
-                cheaty,
-                new Object[] { "   III   ", "  IIIII  ", "  IIXII  ", "  IIIII  ", "   III   ", " nn N nn ",
-                        "nnnnNnnnn", " nn N nn ", "    N    ", 'I', new ItemStack(LudicrousItems.resource, 1, 6), 'X',
-                        new ItemStack(LudicrousItems.resource, 1, 5), 'N', new ItemStack(LudicrousItems.resource, 1, 4),
-                        'n', new ItemStack(LudicrousItems.resource, 1, 3), });
+        if (!Avaritia.isDreamCraftLoaded) {
+            SubTileCheaty.lexicon.setIcon(cheaty);
```

Comment:
Does this line need to be wrapped in the dreamcraft check?

Reply:
Ir not it will show a blank page in the lexicon